Client.xcodeproj/project.pbxproj

```diff
@@ -9967,14 +9968,6 @@
 			path = Fakespot;
 			sourceTree = "<group>";
 		};
-		E1FC23EF2A86290E0089E14D /* Recovered References */ = {
-			isa = PBXGroup;
-			children = (
-				E1516A3A2A7A5746007819A4 /* ReliabilityCardView.swift */,
-			);
-			name = "Recovered References";
-			sourceTree = "<group>";
-		};
 		E1FE132D29C0B334002A65FF /* NotificationSurface */ = {
```

Comment:
I see these things added to my PRs as well, I had to remove them manually. Not sure why they happen.